pipelines/sea-surface-mapping-r/docker/scripts/generate_map.r

```diff
@@ -0,0 +1,55 @@
+library("plotly")
+
+# use scipy to read from netcdf file since R's ncdf4
+# doesn't seem to know how to read it...
+dat <- reticulate::py_run_file("scripts/air-temp-read.py")
```

Comment:
what's the error being thrown? something like:

```R
library(ncdf4)
cdf <- nc_open("./air-temp.nc")
```

should "in theory" work. If not, we can investigate why. Ideally, we'd like to have the entire pipeline in R